README.md

```diff
@@ -6,9 +6,7 @@
 > UniCa$h is a finance tracking application for university students who want to be more financially conscious.
 
 ## About
-
-[//]: # (TODO Change this image)
-![Ui](docs/images/Ui.png)
+![Ui](docs/images/unicash/UniCashWelcome.png)
```

Comment:
Was wondering but why change this image? I thought `Ui.png` was the base image we were using for everything

Reply:
I believe `Ui.png` isn't the most updated. It still contains the "Summary" tab at the top, which we removed from the MainWindow. Anyway, `UniCashWelcome.png` seems to contain everything that `Ui.png` contains, with the exception of the "new" transaction, which I'm not sure we want to showcase.

Reply:
Update: Since Rubesh has updated `Ui.png` in his latest merge, I have reverted this change